src/gordon_gcp/clients/http.py

```diff
@@ -83,9 +78,11 @@ def __init__(self, auth_client=None, session=None):
         if not is_valid:
             await self._auth_client.refresh_token()
 
-    async def request(self, method, url, params=None, body=None,
-                      headers=None, **kwargs):
-        """Make an asynchronous HTTP request.
+    async def request(self, method, url, params=None, body=None, headers=None,
+                      retry_type="none", tries=1, timeout=None,
+                      retry_predicate=None, predicate_args=None,
+                      predicate_kwargs=None, extra_401_retry=True):
+        """Wrapper for _request() that handles retries and backoffs.
```

Comment:
This sort of docstring isn't terribly helpful. Even if it's a wrapper for something, it wraps a (semi) private method, whose docstring can't be assumed would be read (or even written) since it's not a part of the public api. I think the old docstring still makes sense since it's the public API to make an http request.

Reply:
Will fix.